lxd/device/nic_ovn.go

```diff
@@ -148,6 +148,16 @@ func (d *nicOVN) validateConfig(instConf instance.ConfigReader) error {
 		if !dhcpalloc.DHCPValidIP(subnet, nil, net.ParseIP(d.config["ipv4.address"])) {
 			return fmt.Errorf("Device IP address %q not within network %q subnet", d.config["ipv4.address"], d.config["network"])
 		}
+
```

Comment:
Repeat of the questions from the bridged implementation :)

Reply:
So for the ovn implementation, there's no special flags that allow us to continue on when `network.DHCPv4Subnet()` is nil, so we'll error out before my changes.